common/combat_tactics/wc_combat_tactics.txt

```diff
@@ -0,0 +1,1256 @@
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_CAVALRY = 8 } } }
+		}
+        modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SIEGE_EQUIPMENT = 2 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SIEGE_EQUIPMENT = 4 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SIEGE_EQUIPMENT = 6 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SIEGE_EQUIPMENT = 8 } } }
+		}
+	}
+
+	pikemen_offensive = 1
+    dragons_offensive = 1
+    archers_offensive = 1
+    siege_machines_offensive = 1
+    light_infantry_offensive = -1
+    archmage_offensive = -1
+	enemy = {
+		group = harass
+		factor = 1
+	}
+}
+
+wc_magic_volley_tactic = {
+	days = 12
+	sprite = 1
+	group = volley
+
+	trigger = {
+		phase = skirmish
+		OR = {
+			light_infantry = 0.05
+            archmage = 0.05
+		}
+	}
+
+	mean_time_to_happen = {
+		days = 3
+		modifier = {
+			factor = 1.5
+            flank_has_leader = yes
+			leader = {
+				martial = 8
+			}
+			OR = {
+				light_troops = {
+					who = light_infantry
+					value = 0.1
+				}
+				light_troops = {
+					who = archmage
+					value = 0.1
+				}
+			}
+		}
+        modifier = {
+			factor = 1.5
+            flank_has_leader = yes
+			leader = {
+				martial = 12
+			}
+			OR = {
+				light_troops = {
+					who = light_infantry
+					value = 0.2
+				}
+				light_troops = {
+					who = archmage
+					value = 0.2
+				}
+			}
+		}
+		modifier = {
+			factor = 1.5
+            flank_has_leader = yes
+			leader = {
+				martial = 16
+			}
+			OR = {
+				light_troops = {
+					who = light_infantry
+					value = 0.3
+				}
+				light_troops = {
+					who = archmage
+					value = 0.3
+				}
+			}
+		}
+		modifier = {
+			factor = 5
+            flank_has_leader = yes
+			leader = {
+				martial = 8
+				is_magic_leader = yes
+			}
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 2 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 4 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 6 } } }
+		}
+		modifier = {
+			factor = 1.1
+			flank_has_leader = yes
+			leader = { liege = { capital_scope = { TECH_SKIRMISH = 8 } } }
+		}
+	}
+
+	light_infantry_offensive = 1.5
+	archmage_offensive = 1.5
+	archers_offensive = 0.50
+	siege_machines_offensive = 0.50
+	pikemen_offensive = -1.5
+	dragons_offensive = -1.5
+	enemy = {
+		group = harass
+		factor = 1
+	}
+}
+
+wc_bombardment_tactic = {
+	days = 18
+	sprite = 7
+	group = volley
+
+	trigger = {
+		phase = skirmish
+		OR = {
+			archers = 0.05
```

Comment:
Wait a min, even murloc tribe with archers can do bombardment?

Reply:
Si, bombardment can involve any sort of missle